Makefile

```diff
@@ -469,20 +470,16 @@ news/newsflash.inc: $(OMCDATA)/newsflash.txt Makefile
 	    -e 's@: @</td><td class="t">@' \
 	    -e 's@$$@</td></tr>@'
 
-# Make sure we have a copy of vulnerabilities.xml among our public web files
-news/vulnerabilities.xml: $(OMCDATA)/vulnerabilities.xml
-	cp $< $@
-
 # mknews_vulnerability creates two targets and rulesets for creating
 # vulnerability lists for each release.  One target is to create a wrapping
 # HTML file from a template, the other is to create the inclusion file with
 # the actual text.
 #
 # $(1) = output file mod, $(2) = release version switch, $(3) = release version
 define mknews_vulnerability
-news/vulnerabilities$(1).inc: bin/mk-cvepage news/vulnerabilities.xml Makefile
+news/vulnerabilities$(1).inc: bin/cvejsontohtml.py news/secjson Makefile
 	@rm -f $$@
-	./bin/mk-cvepage -i news/vulnerabilities.xml $(2) > $$@
+	python3 ./bin/cvejsontohtml.py -i news/secjson $(2) > $$@
```

Comment:
I'm not at all pleased with an input directory option.  Since we are already figuring out the files here (the `SECJSON_FILES` assignment further down, which will obviously have to be moved up), I suggest this alternative:

``` shell
	python3 ./bin/cvejsontohtml.py $(2) $$(SECJSON_FILES) > $$@
```

and simply let the script use the argument list as the set of files to be parsed.  That way, we have *one* place where we determine what files should be used.  Additionally, the files are visible in the `make` output, which makes it easier to determine if a failure is due to the set of files or something else, when looking at the output in our automation buildbot.

Reply:
agreed (news/vulnerabilities.xml was hardcoded before)

Reply:
Yes, and `news/vulnerabilities.xml` was a single file, so single point of truth, so to say